configs/edgenext/README.md

```diff
@@ -0,0 +1,43 @@
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://github.com/mmaaz60/EdgeNeXt/raw/main/images/EdgeNext.png" width="100%"/>
+</div>
+
+## Results and models
+
+### ImageNet-1k
+
+|        Model         |   Pretrain   | Params(M) | Flops(G) | Top-1 (%) | Top-5 (%) |                              Config                               |                               Download                               |
+| :------------------: | :----------: | :-------: | :------: | :-------: | :-------: | :---------------------------------------------------------------: | :------------------------------------------------------------------: |
+| EdgeNeXt-Base-usi\*  | From scratch |   18.51   |   3.84   |   83.67   |   96.7    | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/edgenext/edgenext-base-usi_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-base_3rdparty-usi_in1k_20220801-909e8939.pth) |
+|   EdgeNeXt-Base\*    | From scratch |   18.51   |   3.84   |   82.48   |   96.2    | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/edgenext/edgenext-base_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-base_3rdparty_in1k_20220801-9ade408b.pth) |
+| EdgeNeXt-Small-usi\* | From scratch |   5.59    |   1.26   |   81.06   |   95.34   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/edgenext/edgenext-small-usi_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-small_3rdparty-usi_in1k_20220801-ae6d8dd3.pth) |
+|   EdgeNeXt-Small\*   | From scratch |   5.59    |   1.26   |   79.41   |   94.53   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/edgenext/edgenext-small_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-small_3rdparty_in1k_20220801-d00db5f8.pth) |
+|  EdgeNeXt-X-Small\*  | From scratch |   2.34    |  0.538   |   74.86   |   92.31   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/edgenext/edgenext-xsmall_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-xsmall_3rdparty_in1k_20220801-974f9fe7.pth) |
+| EdgeNeXt-XX-Small\*  | From scratch |   1.33    |  0.261   |   71.2    |   89.91   | [config](https://github.com/open-mmlab/mmclassification/blob/master/configs/edgenext/edgenext-xxsmall_8xb256_in1k.py) | [model](https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-xxsmall_3rdparty_in1k_20220801-7ca8a81d.pth) |
```

Comment:
Please also add the results to `docs/en/model_zoo.md` and update `docs/en/api/models.rst`